### PR TITLE
Update Understanding and configuring Cloudflare Page Rules (Page Rule…

### PR DESCRIPTION
**Pull request by jeremysimmons**
full url example isn't rendering. 
use literal code markup so that it will render in webpage.

### CODE REVIEW

**Review by ngayerie — APPROVED** [review]

File: content/support/Page Rules/Understanding and configuring Cloudflare Page Rules (Page Rules Tutorial).md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Cloudflare recommends ordering your rules from most specific to least
 specific.
 {{</Aside>}}
 
-A page rule matches a URL pattern based on the following format (comprised of five segments): <scheme>://<hostname><:port>/<path>?<query\_string>
+A page rule matches a URL pattern based on the following format (comprised of five segments): `<scheme>://<hostname><:port>/<path>?<query\_string>`
```
Comment (pedrosousa) [inline content/support/Page Rules/Understanding and configuring Cloudflare Page Rules (Page Rules Tutorial).md]:
```suggestion
A page rule matches a URL pattern based on the following format (comprised of five segments): `<scheme>://<hostname><:port>/<path>?<query_string>`
```
We don't need to escape underscores now that we're using inline code formatting.

**Comment by jeremysimmons** [comment]
@pedrosousa is it possible to bypass the spell-check rule?

**Comment by pedrosousa** [comment]
Yes, I can still merge the PR (once it's ready) even if the spell-checking action fails.

**Comment by jeremysimmons** [comment]
> Yes, I can still merge the PR (once it's ready) even if the spell-checking action fails.

Thanks @pedrosousa anything else you need from me. Thanks for being on this. It thrills me that you allow customers to contribute to the docs.

**Comment by pedrosousa** [comment]
Do you mind committing my suggestion? I can't do it myself in GitHub's web interface (only by checking out the PR locally), since I was the one making the suggestion.

If that option is also not available for you (since the comment has been resolved), I can do it manually.

**Comment by pedrosousa** [comment]
Thank you for the fix, @jeremysimmons!

**Pull request merged**